lib/src/components/NxCollapsibleItems/NxCollapsibleItems.tsx

```diff
@@ -46,11 +46,12 @@ const NxCollapsibleItems: NxCollapsibleItemsFC = function NxCollapsibleItems(pro
         'nx-collapsible-items--empty': isEmpty
       }),
       treeViewId = useUniqueId('nx-collapsible-items', id),
```

Comment:
`treeViewId` is no longer needed

Reply:
Removed here 386ef599cb0a5a2651cf700f4585efb7daec97bf